src/core/core.controller.js

```diff
@@ -101,6 +101,20 @@ function determineLastEvent(e, lastEvent, inChartArea, isClick) {
   return e;
 }
 
+function getDatasetArea(meta) {
+  const {iScale, vScale} = meta;
+  if (iScale && vScale) {
+    const horScale = iScale.isHorizontal() ? iScale : vScale;
+    const verScale = iScale.isHorizontal() ? vScale : iScale;
+    return {
+      left: horScale.left,
+      right: horScale.right,
+      top: verScale.top,
+      bottom: verScale.bottom
+    };
+  }
```

Comment:
This is what I was suggesting: 

```suggestion
  const {xScale, yScale} = meta;
  if (xScale && yScale) {
    return {
      left: xScale.left,
      right: xScale.right,
      top: yScale.top,
      bottom: yScale.bottom
    };
  }
```

Reply:
yes, already committed